README.md

```diff
@@ -117,18 +141,6 @@ Cookies.remove('name', { path: '', domain: '.yourdomain.com' });
 
 *Note: Removing a nonexistent cookie does not raise any exception nor return any value.*
 
-## Namespace conflicts
-
-If there is any danger of a conflict with the namespace `Cookies`, the `noConflict` method will allow you to define a new namespace and preserve the original one. This is especially useful when running the script on third party sites e.g. as part of a widget or SDK.
-
-```javascript
-// Assign the js-cookie api to a different variable and restore the original "window.Cookies"
-var Cookies2 = Cookies.noConflict();
-Cookies2.set('name', 'value');
-```
-
-*Note: The `.noConflict` method is not necessary when using AMD or CommonJS, thus it is not exposed in those environments.*
-
```

Comment:
Wait, we can bring back this piece of documentation, and mention this applies to the UMD module..